flink/v1.12/flink/src/main/java/com/netease/arctic/flink/read/FlinkKafkaConsumerBase.java

```diff
@@ -910,11 +910,9 @@ private void createAndStartDiscoveryLoop(AtomicReference<Exception> discoveryLoo
                 // soon as possible
 
                 while (running) {
-                  if (LOG.isDebugEnabled()) {
-                    LOG.debug(
-                        "Consumer subtask {} is trying to discover new partitions ...",
-                        getRuntimeContext().getIndexOfThisSubtask());
-                  }
+                  LOG.debug(
+                      "Consumer subtask {} is trying to discover new partitions ...",
+                      getRuntimeContext().getIndexOfThisSubtask());
```

Comment:
I'm not sure if we can consider the cost of the code `getRuntimeContext().getIndexOfThisSubtask()` here to be sufficiently low.

This class was copied from the Flink community and is deprecated in the Arctic. I suggest not making any modifications to it.

Reply:
I agree with you, done.